### ISSUE TITLE
Return pivot point of selection tool in level strip

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#2121]
### Issue Summary

![pivot point selection tool](https://user-images.githubusercontent.com/29790512/42134646-4a2da93c-7d40-11e8-9409-adfebfd94ce1.png)

### Steps to Reproduce

select any stroke with selection tool in level strip

### Expected Results
Pivot point enableb too

### System Information
w10
* **OpenToonz Version**: 
all
* **Operating System**: 
* **CPU**: 
* **Memory**: 
* **Graphics Card**: 
* **Graphics Tablet**:

**Comment by openanim** [comment]
Ah! good new thank you, I will try different shades of gray for that then.I close the problem.

**Comment by RodneyBaker** [comment]
Concur with @manongjohn in that the pivot is present but very hard to see and manipulate.  The fix would seem to be to adjust it in such a was as to make it stand out.

Aside:  In the far flung future it'd be nice if the pivot could remain at the location it is moved to (i.e. sticky) and that offset exposed via UI so that it could be reset to zero (0) offset.  I believe there was/is a feature discussion related to that already but mention it again here as it relates to the pivot/offset under consideration.

**Comment by openanim** [comment]
I agree that it should be fixed after moving it, but the current temporary behavior should be available as well as useful.

**Comment by manongjohn** [comment]
This reminds me of an annoying issue i used to have with the pivot point resetting back to center after moving the stroke.

**Comment by openanim** [comment]
It is clear that for the cut-out in particular it would be very beneficial to have this additional option with the use of groups.

**Comment by RodneyBaker** [comment]
Nice thoughts.  @manongjohn The thought about Groups maintaining the changed pivot point is a very good one.  I need to remember that one.  The benefit to that idea is that Groups.. which are simply containers for any selection are an abstraction and exposing Groups so that the user can organize and manipulate them has long been on my radar.  

An interesting thing about Groups (or what I would call Named Groups) also is that the user can group the exact same things multiple times and have settings for one Group that are different from other Groups.  So the example with pivots is that the user could create two Groups containing the exact same things and have only the pivots different.  Then (in that world where Named Groups exist) the user could choose between Groups to access the desired pivot.  If they just wanted to update the position of the pivot they would move the pivot, save the Group (just by exiting the group.. not actually saving anything manually) and the setting would be updated. 

I'm use to Named Groups in another program and it's very useful.
Grouping is an abstraction that allows an added layer of data normall reserved for animated objects but... without resorting to actual keyframing or animation.  Not to dive too deeply into that but if I understand what is going on correctly Groups are basically the template that keyframes later use.  While it's a little beyond the immediate need here.. by allowing Groups to store the pivot point/center offset that pivot could even later be keyframed/animated.

**Comment by RodneyBaker** [comment]
This won't be self evident but I post this again because it relates to the idea of the Group maintaining the pivot offset.

I discovered a way that currently in OT we can in fact adjust the pivot point and then have that adjusted position saved in the Group.  This is an extreme hack of little to no real use... it's more of a Proof of concept that Groups in OT are the likely place where this pivot offset data would be stored, retrieved, edited and 'saved.  Interestingly, the change in pivot offset is saved simply by Right Clicking and selecting 'Exit Group'.

In this image basically there are two copies of the same rectangle.  For demo purposes I've added a second object (shown as a circle with dotted lines) that forces the pivot's location by changing the size of the Group itself... because currently a pivot will allows be located at the exact center of all the objects included in the Group.  On the left (Group 1) the pivot is forced to the right.  On the Right (Group 2) the pivot remains in the center.  To move the pivot location the helper shape is just moved to where it is needed to have the pivot center where desired to be located.   To save the location Right Click > Exit Group.

Obviously this is not an acceptable way to adjust the pivot.  It's just a proof of the concept of Groups having a set of 'pivot offset properties' that get saved with the Group.  ;)
And that would be the suggestion for the far flung future (besides exposing the Groups in the UI so the user can see, name and rename and organize them)... to have a Group Properties menu.

![savepivotoffsetviagroupproofofconcept](https://user-images.githubusercontent.com/2576432/42140029-32b9d8b6-7d5e-11e8-814a-fe693620ba3b.png)
